### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/polkit/commit/7183c3c9607aee2bc3536e58b4937b0b11924232#commitcomment-52099084

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/polkit/commit/5fc57193a7b705e1c671052c03b7dab5a6fcd87d

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -25,7 +25,7 @@ Changes since polkit 0.118:
  Inigo Martinez:
     meson build system added alongside autotools (many thanks!!)
 
- Matthias Classen:
+ Matthias Clasen:
```
Comment (jsoref) [inline NEWS:28]: https://github.com/freedesktop/polkit/search?q=Matthias+Clasen

File: NEWS [hunk 2]
```diff
@@ -863,7 +863,7 @@ David Zeuthen (65):
       docs: update SEE ALSO to make each man page point to all other man pages
       Clarify docs a bit
       polkitd: add reference to polkit(8) from its man page
-      Fix speling
+      Fix spelling
```
Comment (jsoref) [inline NEWS:866]: I know, it's cute. If you want to keep it, I can drop the change.

I left two news items for correcting misspellings. Personally, I don't see value in being told what the incorrect forms of words were. If it's a problem, when I compile, I'll figure it out.

File: data/org.freedesktop.PolicyKit1.Authority.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
   <interface name="org.freedesktop.PolicyKit1.Authority">
     <annotation name="org.gtk.EggDBus.DocString.Summary" value="Authority Interface"/>
-    <annotation name="org.gtk.EggDBus.DocString" value="This D-Bus interface is implemented by the <literal>/org/freedesktop/PoliycKit1/Authority</literal> object on the well-known name <literal>org.freedesktop.PolicyKit1</literal> on the system message bus."/>
+    <annotation name="org.gtk.EggDBus.DocString" value="This D-Bus interface is implemented by the <literal>/org/freedesktop/PolicyKit1/Authority</literal> object on the well-known name <literal>org.freedesktop.PolicyKit1</literal> on the system message bus."/>
```
Comment (jsoref) [inline data/org.freedesktop.PolicyKit1.Authority.xml:7]: A typo of this form was listed in news as corrected. I presume this instance isn't intentionally misspelled.

File: docs/man/pkcheck.xml [hunk 4]
```diff
@@ -89,7 +89,7 @@
 
   </refsynopsisdiv>
 
-  <refsect1 id="pkcheck-descsription">
+  <refsect1 id="pkcheck-description">
```
Comment (jsoref) [inline docs/man/pkcheck.xml:92]: This could break external references

File: docs/man/pkttyagent.xml [hunk 5]
```diff
@@ -66,7 +66,7 @@
 
   </refsynopsisdiv>
 
-  <refsect1 id="pkttyagent-descsription">
+  <refsect1 id="pkttyagent-description">
```
Comment (jsoref) [inline docs/man/pkttyagent.xml:69]: Ibid

File: src/polkitagent/polkitagenthelperprivate.h [hunk 6]
```diff
@@ -17,7 +17,7 @@
  * Boston, MA 02110-1301, USA.
  *
  * Authors: David Zeuthen <davidz@redhat.com>,
- *          Andrew Psaltis <ampsalits@gmail.com>
+ *          Andrew Psaltis <ampsaltis@gmail.com>
```
Comment (jsoref) [inline src/polkitagent/polkitagenthelperprivate.h:20]: https://github.com/freedesktop/polkit/search?q=ampsalits%40gmail.

**Comment by fdo-mirror** [comment]
You have submitted a GitHub pull request to a freedesktop.org repository. freedesktop.org's software is developed elsewhere - usually on [GitLab](https://gitlab.freedesktop.org). This GitHub repository is a read-only mirror. Please resubmit your request within GitLab. The project should include a CONTRIBUTING file explaining how to best submit your contributions to us. This pull request is being closed automatically. Thankyou for your work, and we look forward to seeing your contributions.

**Pull request closed without merging**